GameServerLib/GameObjects/Spells/Buff.cs

```diff
@@ -114,15 +114,6 @@ public Buff(Game game, string buffName, float duration, int stacks, ISpell origi
         public void ActivateBuff()
         {
             _buffGameScript.OnActivate(TargetUnit, this, OriginSpell);
-            if (!OriginSpell.SpellData.CantCancelWhileChanneling)
```

Comment:
The problem with the recall should be reopened in the issues. I do agree that this code in its current state has a lot of side effects and so it is fine to remove it till it get fixed.